openpower/package/petitboot/petitboot.mk

```diff
@@ -19,6 +19,8 @@ PETITBOOT_CONF_OPTS += --with-ncurses --without-twin-x11 --without-twin-fbdev \
 	      HOST_PROG_SHUTDOWN=/usr/libexec/petitboot/bb-kexec-reboot \
 	      $(if $(BR2_PACKAGE_BUSYBOX),--with-tftp=busybox)
 
+PETITBOOT_AUTORECONF_ENV += PETITBOOT_VERSION=`cat $(PETITBOOT_VERSION_FILE) | cut -d '-' -f 2-`
```

Comment:
The buildroot variable ${PETITBOOT_VERSION} is just "v1.2.1" for example, whereas the version file includes the full "v1.2.1-(opdirty?)-(patchversion)" string.
Skiboot at least does something similar here: https://github.com/open-power/op-build/blob/master/openpower/package/skiboot/skiboot.mk#L30